plugins/agents/muninpluginrunner/example.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```

Comment:
We could show a good example and start the script by

```bash
#!/bin/bash

set -euo pipefail
```

Reply:
@abrander added + a better oneliner for getting data :)